main.js

```diff
@@ -0,0 +1,14 @@
+let allPosts = document.querySelector("header nav ul li");
+allPosts.remove();
+
+let featuredTime = document.querySelector(".featured span");
+featuredTime.remove();
+
+let lastPost = document.querySelector("main .posts article");
+lastPost.remove();
```

Comment:
Although this does pass the tests, you're not actually accessing the **last** element, you're accessing the **first** element.
```suggestion
let allPosts = document.querySelectorAll("main .posts article");
let lastPost = allPosts[allPosts.length - 1]
lastPost.remove()
```
`document.querySelector` actually grabs the **first** matching element, NOT the **last** one. In order to grab the last element out of multiples of the same type of element, you have to use `querySelectorAll`. `querySelectorAll `returns a **node list**, which is similar to an array, so you can get the last element in the same way that you would for an array: `allPosts[allPosts.length - 1]`